src/plugins/vis_type_vega/public/expressions/line_vega_spec_fn.ts

```diff
@@ -133,14 +149,16 @@ export const buildYAxis = (
   valueAxis: ValueAxis,
   visParams: VisParams
 ) => {
+  const subAxis = {
+    title: cleanString(valueAxis.title.text) || column.name,
+    grid: visParams.grid.valueAxis !== undefined,
+    orient: valueAxis.position,
+    labels: valueAxis.labels.show,
+    labelAngle: valueAxis.labels.rotate,
+  };
+  if (column.meta?.type === 'percentile_ranks') Object.assign(subAxis, { format: '.0%' });
```

Comment:
Can you add a comment about why this is needed? are there other types that could potentially need other formatting too?

Reply:
done